### ISSUE TITLE
Add ability to enter time remaining and the option to use it to generate burndown charts

### ISSUE THREAD

**Issue by cckozie** [ZenHubIO/support#637]
I'm new to ZenHub but one feature that I believe is missing is the ability to enter and update the estimated time remaining (hours, not points) on each issue or task. In my experience that data gives us the most accurate information about where we are in the process of completing the issues and the earliest warnings on where we may be in trouble. I see that this was part of the request made in issue #133, but it was closed as being part of #69. I don't believe that #69 satisfies this requirement, but I could be wrong.

**Comment by azenMatt** [comment]
Hi @cckozie - agreed that this is not captured by #69 - lets keep this option open to track it :+1:

**Comment by alecmerdler** [comment]
I agree, currently the burndown chart is only updated when issues are CLOSED in Github, whereas I believe the true utility of the burndown chart is to reflect the total story points/hours remaining in the project. In our workflow, an issue can only be closed after it has been QA'd, and our true progress in term of effort remaining isn't reflected on the burndown chart.

**Comment by digitalhoax** [comment]
+1

**Comment by arunachalam1978** [comment]
any update on this feature ?

**Comment by marcusg** [comment]
+1

**Comment by profgmcmahon** [comment]
Hi,

This feature is central to creating an accurate and meaningful sprint burndown chart for scrum teams.
I think this would be a really great improvement.

Is this enhancement being considered for the near future?

**Comment by vincentmarceljeanbertin** [comment]
+1

**Comment by juliana-hsu** [comment]
We use the estimate field to show the size of work but we need another field to track the progress of completion against the estimate to confirm if things are on track.  Is there any roadmap plan for this feature?
